sql-plugin/src/main/301until330-all/scala/com/nvidia/spark/rapids/shims/v2/GpuRangePartitioning.scala

```diff
@@ -14,34 +14,22 @@
  * limitations under the License.
  */
 
-package com.nvidia.spark.rapids
+package com.nvidia.spark.rapids.shims.v2
 
-import com.nvidia.spark.rapids.shims.v2.ShimExpression
+import com.nvidia.spark.rapids.GpuRangePartitioningBase
```

Comment:
Should generate a compile error since the class was removed.  Was this built locally before posting?